disnake/guild.py

```diff
@@ -1829,6 +1831,10 @@ async def edit(
         community: :class:`bool`
             Whether the guild should be a Community guild. If set to ``True``\\, both ``rules_channel``
             and ``public_updates_channel`` parameters are required.
+        invites_disabled: :class:`bool`
+            Whether the guild can be joined.
```

Comment:
Similar to other parameters, this should note that it requires the community feature to be enabled.

Reply:
it didn't require that when I initially tested it :^)

Reply:
https://github.com/DisnakeDev/disnake/pull/718/commits/ae7b0b6b8891a95f89fd8ccc9f803a789f44b61c

Reply:
Reopening this since this limitation may be due to the experiment filter, and not actually a requirement once fully rolled out. Suppose we'll just have to wait and see :thinking:

Reply:
It is now fully out, but only to community guilds, so..